### DISCUSSION TITLE
Unable to run configure.exe

### DISCUSSION THREAD

**Discussion by ptiruvaipati** [ImageMagick/ImageMagick#7278] [Help] [answered]
### ImageMagick version

7.1.1-31

### Operating system, version and so on

Windows 10

### Description

Hi Team,

We are planning to upgrade ImageMagick to 7.1.1-31.

I have followed the below link to build ImageMagick for Windows.

https://imagemagick.org/script/advanced-windows-installation.php

It's throwing the error when I tried to run configure.exe (VisualMagick/configure)

Here I attached the screenshot of the error.
![image](https://github.com/ImageMagick/ImageMagick/assets/123535772/34667e7c-710d-42ea-98ee-d18e5bb4c565)

We are using Visual Studio 2019.

I tried to run configure.exe in the debug mode from command line. 
It's throwing the error like "**Unable to find version file for: aom**". 

I am getting the same error with ImageMagick 7.1.1-30. 

Appreciate your help. 

Regards,
Pavan

**Comment by dlemstra** [comment, marked as answer]
The VisualMagick repository has been archived and we should update that page. You can find the new information here: https://github.com/ImageMagick/ImageMagick-Windows

**Reply by ptiruvaipati** [reply]
Thanks @dlemstra for your quick help.
